data/quickstarts/oneapi-aikit-notebook.yaml

```diff
@@ -87,7 +87,7 @@ spec:
       review:
         instructions: |-
           #### Verify that the sample notebook ran successfully
-          Did all cells of the notebook run successfully and printed out the statement "[CODE\_SAMPLE\_COMPLETED\_SUCCESFULLY]" at the end?
+          Did all cells of the notebook run successfully and printed out the statement "[CODE_SAMPLE_COMPLETED_SUCCESFULLY]" at the end?
```

Comment:
the escape char is needed here. It was added in the PR https://github.com/red-hat-data-services/odh-dashboard/pull/222/files

Reply:
Got it, thanks Berto